src/Microsoft.AspNet.Http.Abstractions/QueryString.cs

```diff
@@ -231,7 +231,7 @@ public override bool Equals(object obj)
 
         public override int GetHashCode()
         {
-            return (_value != null ? _value.GetHashCode() : 0);
+            return _value?.GetHashCode() ?? 0;
```

Comment:
Was just saying how it could work; never actually looked at what it did do in depth before